libs/execution/test/utils/test-infrastructure-util.ts

```diff
@@ -55,13 +57,19 @@ export function getTestExecutionContext(
     'pipelines@0',
   ) as PipelineDefinition;
 
+  const operatorEvaluatorRegistry = new DefaultOperatorEvaluatorRegistry();
+
   const executionContext = new ExecutionContext(
     pipeline,
     new TestExecExtension(),
     new DefaultConstraintExtension(),
     new CachedLogger(runOptions.isDebugMode, undefined, loggerPrintLogs),
+    new WrapperFactory(operatorEvaluatorRegistry),
     runOptions,
-    new EvaluationContext(new RuntimeParameterProvider()),
+    new EvaluationContext(
+      new RuntimeParameterProvider(),
+      operatorEvaluatorRegistry,
+    ),
```

Comment:
This code snippet exists at a plethora of places. Can we move to a method?

Reply:
Done in [8db76f0](https://github.com/jvalue/jayvee/pull/536/commits/8db76f02389044292398c2d96001be9ca9d03657).